docs/modules/deployment/pages/restore/centos-rhel8/restore-centos-rhel8.adoc

```diff
@@ -0,0 +1,78 @@
+[[restore-centos-rhel8]]
+.Restore User/Group definition
+
+[source, console]
+----
+sudo cat /tmp/opennms-passwd.txt >> /etc/passwd && \
+sudo cat /tmp/opennms-group.txt >> /etc/group
+----
+
+.Create log folder
+[source, console]
+----
+sudo mkdir /var/log/opennms && \
+chown opennms. /var/log/opennms
+----
+
+.Restore OpenNMS {page-component-title} binaries, RRD archives, configurations
+
+[source, console]
+----
+sudo tar -xzf /tmp/opennms-opt.tar.gz -C / && \
+sudo tar -xzf /tmp/opennms-var.tar.gz -C /
+----
+
+.Create OpenNMS systemd unit
+[source, console]
+----
+sudo cp /opt/opennms/etc/opennms.service /lib/systemd/system/
+----
+
+._Assign CAP_NET_RAW capabilities_
+
+Follow the xref:deployment:core/centos-rhel/initialize-core.adoc#initialize-core-cap[Assign CAP_NET_RAW capabilities] section.
+
+.Install PostgreSQL Database
+[source, console]
+----
+sudo dnf -y install postgresql-server postgresql
+----
+
+.Change the access policy for PostgreSQL
+[source, console]
+
+Follow the xref:deployment:core/centos-rhel8/postgresql.adoc#core-centos-rhel8-pg_hba[pg_hba] config section.
+
+.Restore OpenNMS Database
+[source, console]
+----
+sudo runuser -l postgres -c 'psql -f /tmp/opennms_postgres.dmp postgres'
+----
+
+.Install OpenJDK
+[source, console]
+----
+sudo dnf -y install java-11-openjdk
+sudo /opt/opennms/bin/runjava -s
+----
+
+.Run `install -dis` command
+[source, console]
+----
+sudo /opt/opennms/bin/install -dis
+----
+
+.Firewall Configuration
+[source, console]
+
+Depending on where OpenNMS gets restored the firewall settings have to be set again.
+
+xref:deployment:core/centos-rhel/initialize-core.adoc#initialize-core-firewall
```

Comment:
Referencing now to the install guide, to avoid duplication.
Can we just put those ref links here like this to get rendered properly?